apache/superset
# chore(translations): Slovenian translation update

### SUMMARY
messages.po file for Slovenian (sl_SI) translation is updated (I updated .po file with babel_update.sh script, added missing translations, fixed fuzzy translation)


### TESTING INSTRUCTIONS
Change language settings to Slovenian and check translations via GUI or manually check them in messages.po file.

### ADDITIONAL INFORMATION
